src/screens/VisualTests/StoryInfo.tsx

```diff
@@ -92,7 +92,9 @@ export const StoryInfo = ({
     details = (
       <Text>
         <b>
-          {changeCount ? pluralize("change", changeCount, true) : "No changes"}
+          {changeCount && status === TestStatus.Pending
+            ? pluralize("change", changeCount, true)
+            : "No changes"}
```

Comment:
I know this was what was requested but it doesn't quite make sense to me. The change doesn't go away when it is accepted?

Should it be still there for denied btw?

Reply:
Yeah I felt the same way. "Accepted 1 change" would be more sensible, I think. @MichaelArestad WDYT?

Reply:
Those are good points. The idea was to say there are no changes left for a user to review. Do you have a better idea as to what we could show here?

Reply:
"Accepted 1 change" or just "Changes accepted" are both good options I think.

Reply:
Will it show "Changes accepted" the next time they arrive at this view? Will other users see this?

Reply:
I think the simplest thing would be for everyone to see the same thing.